File: Hackerrank/day01/main.cs
using System;
using System.Collections.Generic;
using System.IO;

class Solution {
    static void Main(String[] args) {
        int i = 4;
        double d = 4.0;
        string s = "HackerRank ";

        
        // Declare second integer, double, and String variables.
        int entero;
        double flotante;
        string cadena;
        // Read and save an integer, double, and String to your variables.
        entero = int.Parse(Console.ReadLine());
        flotante = double.Parse(Console.ReadLine());
        cadena = Console.ReadLine();
        // Print the sum of both integer variables on a new line.
        Console.WriteLine(entero + i);
        // Print the sum of the double variables on a new line.
        double result1 = flotante + d; 
        Console.WriteLine($"{result1:F1}");
        // Concatenate and print the String variables on a new line
        // The 's' variable above should be printed first.
        string result = $"{s}{cadena}";
        Console.WriteLine(result);

    }
}

File: Hackerrank/day01/main.pl
$i = 4;
$d = 4.0;
$s = 'HackerRank ';
# Declare second integer, double, and String variables.
my $entero = <STDIN>;
my $flotante = <STDIN>;
my $cadena = <STDIN>;
# Read and save an integer, double, and String to your variables.

# Print the sum of both integer variables on a new line.
my $result = $entero + $i;
print "$result\n";
# Print the sum of the double variables on a new line.
my $result1 = $flotante + $d;
printf ("%.1f\n", $result1);
# Concatenate and print the String variables on a new line
# The 's' variable above should be printed first.
my $result2 = $s . $cadena;
print $result2;
